mobile/src/main/res/xml/item_update_widget.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appwidget-provider xmlns:android="http://schemas.android.com/apk/res/android"
+                    android:minWidth="40dp"
+                    android:minHeight="40dp"
+                    android:updatePeriodMillis="0"
+                    android:previewImage="@drawable/item_update_widget_preview"
+                    android:initialLayout="@layout/widget_item_update"
+                    android:widgetCategory="home_screen|keyguard"
```

Comment:
I know homescreen widgets aren't used anymore, but for the Android versions that *have* homescreen widgets, this one doesn't seem suitable to place there.

Reply:
Do you mean keyguard? IMO it's practical to quickly turn on lights, for example.

Reply:
@maniac103 This is still unresolved.

Reply:
Yes, I meant keyguard. I still think only information widgets should be placed there, but I don't have a strong opinion on this one, so it's fine.